packages/password-input/src/PasswordInput/PasswordInput.tsx

```diff
@@ -123,30 +135,32 @@ export const PasswordInput = React.forwardRef<
                     inputDisabledBaseStyles,
                     inputDisabledThemeStyles[theme],
                   )]: disabled,
-                  [inputIconSizeStyles[size]]:
-                    !hasNotifications && state !== State.None,
                 },
               )}
               onChange={handleChange}
               readOnly={disabled}
               {...rest}
             />
-            {/* If a custom message container is used, an icon will render inside the input to represent the state of the input. In the case that stateNotification === `none`, no icon will appear. */}
-            {!hasNotifications && state !== State.None && (
-              <InputIcon state={state} size={size} disabled={disabled} />
-            )}
             <PasswordToggle
               showPassword={showPassword}
               handlePasswordToggleClick={handlePasswordToggleClick}
               size={size}
             />
           </div>
-          {hasNotifications && (
```

Comment:
This was only hidden if a custom container was used, which is why getting rid of `aria-live` didn't matter. Since we will always show a message, even if the consumer uses a custom container, we should keep a div with `aria-live` on the page.

Reply:
updated!